lib/scrabble/dictionary.rb

```diff
@@ -1,4 +1,17 @@
 module Scrabble
 	class Dictionary
+
+		attr_accessor
```

Comment:
Hmm looks like this is here unnecessarily since there are no attributes being used